packages/frontend/src/hooks/fungibleTokensIncludingNEAR.js

```diff
@@ -28,7 +28,7 @@ export const useFungibleTokensIncludingNEAR = function () {
     );
 
     useEffect(() => {
-        setFungibleTokensList(fungibleTokensIncludingNEAR(tokens, balanceToDisplay, nearTokenFiatValueUSD));
+        setFungibleTokensList(fungibleTokensIncludingNEAR(tokens.sort((a, b) => a.name.localeCompare(b.name)), balanceToDisplay, nearTokenFiatValueUSD));
```

Comment:
Would it make sense to move the `.sort()` to the `tokens` declaration on line 22 so that the initial state also gets the sorted list?

Reply:
Great point:)